AzureCalling/External/Calling/CallingContext.swift

```diff
@@ -183,6 +196,18 @@ class CallingContext: NSObject {
         }
     }
 
+    func pauseVideo() {
+        if isCameraPreferredOn {
+            stopVideo { [weak self] _ in
+                guard let self = self else {
+                    return
+                }
+                self.isVideoOnHold = true
+                print("Local video paused successfully")
+            }
+        }
+    }
+
```

Comment:
I think it would be better to rename those functions to more specific name
-startLocalVideoStream
-stopLocalVideoStream
-pauseLocalVideoStream
-resumeLocalVideoStream